Add ui 2

smirecki opened the pull request:
I've added a web frontend with Express and Node, templating with Handlebars, and style with Material Design Lite.

smirecki committed:
Changed logic to correct output

Update to avoid strange output or hacks.

smirecki committed:
Merge remote-tracking branch 'upstream/master'

Conflicts:
	index.js

smirecki committed:
Add UI branch

Building a UI for the failed tweets commandline app

smirecki committed:
Merge pull request #1 from smirecki/add-ui

Add UI branch

smirecki committed:
Add to packages.json

Added to packages.json, to use Express.js and Handlebars.

smirecki committed:
Merge branch 'Add-UI-2' of https://github.com/smirecki/failed-tweets into Add-UI-2

smirecki committed:
Pre-Alpha UI build - Update index.js

Adding Express.js and Handlebars

smirecki committed:
Handlebars logic add

Adding folders and files for Handlebars build style.

smirecki committed:
minor fix

Github merge change text fix

smirecki committed:
Built ES5 version

Built ES5 version of recent change, pre-alpha UI build.

smirecki committed:
Fix index.js

Corrected a few errors

smirecki committed:
Rebuilt ES5 index.js

I had to rebuild the ES5 version after the corrections.

smirecki committed:
Removed UI folder

Removed unused UI folder.

smirecki committed:
README update added Output notes

I add info about the what to expect for output and notes on the newer web ouptu.

smirecki committed:
Correcting Github Markdown for README

They to correct the backticks for proper code block styling on this file.

smirecki committed:
Updated UI with tables

The UI has been updated to use tables and the look has been given minor improvements.

smirecki committed:
Updated the UI/UX with Google's MDL

The UI/UX is now styled with Google's MDL, which improves the look and feel of the app.
This change effect the style with css and visual feed back with javascript.  MDL also makes the page adjust better to different screen sizes.

smirecki committed:
Adding image of UI for README section

I've added a folder for images and included an image of the UI for the README document.

smirecki committed:
updated README to include image

Added an image to the README file.

smirecki committed:
Merge remote-tracking branch 'upstream/master'

smirecki committed:
Merge remote-tracking branch 'upstream/master' into Add-UI-2

Conflicts:
	build/index.js
	index.js

smirecki committed:
Changes to UI

Added code to make it function better.

smirecki committed:
Removed old folder and file

Removed old content that wasn't used.

smirecki committed:
Merge remote-tracking branch 'origin/master' into Add-UI-2

smirecki committed:
Update README file

I changed the wording on the README.  Material Design Lite is not made by Google, Material Design is a standard created by them and Lite is an implementation of it.

rkulla commented on build/index.js:
```diff
@@ -1,5 +1,20 @@
+
+app.use(express['static'](__dirname + '/public'));
+
+'use strict';
```
Please put `'use strict';` at the very top of the file

smirecki committed:
Add 'use strict' per @rkulla request

I've added 'use strict' to the begining of this file, per the request of @rkulla in the notes.

smirecki committed:
Reworked output section to be more DRY

This code was changed to be more DRY for the CLI and HTML array, now that they both use the same array data.

smirecki committed:
Fix URL output typo error

I corrected an typo that was causing an output error in the URL string.

smirecki committed:
Make URL strings links to tweets

The URL strings are now clickable links to tweets.

smirecki committed:
Add favicons folder and favicons

Added a favicons folder and favicons for site.

smirecki committed:
Removed unneeded image

I removed an unneeded image and the folder that contained it.

smirecki committed:
UI make text URL clickable, slight UX redesign, and add date

I've made the URL of the tweet clickable by making the Tweet Text clickable to the Tweet URL.  I also added the date of the tweet.  Also the UI/UX has been changed: colors, hover state, and background colors.

smirecki committed:
Slight color changes

I've made some slight color changes to the table date background and link hover state.
